pkg/rtc/types/interfaces.go

```diff
@@ -116,6 +116,11 @@ type MediaTrack interface {
 	Source() livekit.TrackSource
 	IsSimulcast() bool
 
+	ParticipantID() livekit.ParticipantID
+	ParticipantIdentity() livekit.ParticipantIdentity
```

Comment:
Called these `ParticipantX`, maybe `PublisherX` is a more appropriate name? Please let me know.

Reply:
Agreed `PublisherX` seems better, since we make use of `SubscriberX` in this class

Reply:
Cool. Will change it.